dotnet/src/VectorDataIntegrationTests/VectorDataIntegrationTests/EmbeddingGenerationTests.cs

```diff
@@ -370,6 +383,15 @@ public class RecordWithCustomerVectorProperty
         public string? Text { get; set; }
     }
 
+    public class RecordWithReadonlyMemoryOfFloatVectorProperty
```

Comment:
nit: Small renaming based on `ReadOnlyMemory` type.
```suggestion
    public class RecordWithReadOnlyMemoryOfFloatVectorProperty
```